react-nd/src/hooks/use-fetch.jsx

```diff
@@ -0,0 +1,41 @@
+import { useState, useEffect } from "react";
+import data from '../../data/db.json';
+
+const useFetchFile = () => {
```

Comment:
Geriau turėti 2 atskirus konkrečius hooksus (kurie perpanaudotų abstraktų hook'są), kurie perpanaudotų bendrą logiką. Dabar gaunasi, kad norint panaudoti service'us parsitraukiate iš failo ir business'us, kas yra neperspektyvu (not expandable, not scalable)